uvloop/loop.pyx

```diff
@@ -186,6 +186,8 @@ cdef class Loop:
 
         # Set to True when `loop.shutdown_asyncgens` is called.
         self._asyncgens_shutdown_called = False
+        # Set to True when `loop.shutdown_default_executor` is called.
+        self._executor_shutdown_called = False
```

Comment:
I think you should also add the `_executor_shutdown_called` field to loop.pxd.

Reply:
Yep, that would make sense, seeing as `_asyncgens_shutdown_called` is also present there. I'm just not entirely used to working with languages that use structs to specify the fields in header files. :-)

Reply:
Yeah, Cython looks a bit alien at first